[0.220s] Invoking command in '/home/dynepanch/ros2_ws3/ros2_ws/build/person_msgs': /usr/bin/cmake /home/dynepanch/ros2_ws3/ros2_ws/person_msgs -DCMAKE_INSTALL_PREFIX=/home/dynepanch/ros2_ws3/ros2_ws/install/person_msgs
[0.232s] -- Found ament_cmake: 1.3.2 (/opt/ros/humble/share/ament_cmake/cmake)
[0.361s] -- Found rosidl_default_generators: 1.2.0 (/opt/ros/humble/share/rosidl_default_generators/cmake)
[0.366s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[0.375s] -- Found rosidl_adapter: 3.1.3 (/opt/ros/humble/share/rosidl_adapter/cmake)
[0.542s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[0.751s] [31mCMake Error at /opt/ros/humble/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:59 (message):
[0.752s]   execute_process(/usr/bin/python3.10 -m rosidl_adapter --package-name
[0.752s]   person_msgs --arguments-file
[0.752s]   /home/dynepanch/ros2_ws3/ros2_ws/build/person_msgs/rosidl_adapter__arguments__person_msgs.json
[0.752s]   --output-dir
[0.752s]   /home/dynepanch/ros2_ws3/ros2_ws/build/person_msgs/rosidl_adapter/person_msgs
[0.753s]   --output-file
[0.753s]   /home/dynepanch/ros2_ws3/ros2_ws/build/person_msgs/rosidl_adapter/person_msgs.idls)
[0.753s]   returned error code 1:
[0.753s] 
[0.753s]   Error processing 'unit8 age' of 'person_msgs/Person': ''unit8' is an
[0.754s]   invalid message name.  It should have the pattern '^[A-Z][A-Za-z0-9]*$''
[0.754s] 
[0.754s]   Traceback (most recent call last):
[0.754s] 
[0.754s]     File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
[0.754s]       return _run_code(code, main_globals, None,
[0.755s]     File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
[0.755s]       exec(code, run_globals)
[0.755s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__main__.py", line 19, in <module>
[0.755s]       sys.exit(main())
[0.756s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/main.py", line 53, in main
[0.756s]       abs_idl_file = convert_to_idl(
[0.756s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__init__.py", line 19, in convert_to_idl
[0.756s]       return convert_msg_to_idl(
[0.756s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/msg/__init__.py", line 28, in convert_msg_to_idl
[0.757s]       msg = parse_message_string(package_name, input_file.stem, content)
[0.757s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 520, in parse_message_string
[0.757s]       Type(type_string, context_package_name=pkg_name),
[0.757s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 277, in __init__
[0.757s]       super(Type, self).__init__(
[0.758s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 205, in __init__
[0.758s]       raise InvalidResourceName(
[0.758s] 
[0.758s]   rosidl_adapter.parser.InvalidResourceName: 'unit8' is an invalid message
[0.759s]   name.  It should have the pattern '^[A-Z][A-Za-z0-9]*$'
[0.759s] 
[0.759s] Call Stack (most recent call first):
[0.759s]   /opt/ros/humble/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
[0.759s]   CMakeLists.txt:12 (rosidl_generate_interfaces)
[0.759s] 
[0.760s] [0m
[0.760s] -- Configuring incomplete, errors occurred!
[0.760s] See also "/home/dynepanch/ros2_ws3/ros2_ws/build/person_msgs/CMakeFiles/CMakeOutput.log".
[0.769s] Invoked command in '/home/dynepanch/ros2_ws3/ros2_ws/build/person_msgs' returned '1': /usr/bin/cmake /home/dynepanch/ros2_ws3/ros2_ws/person_msgs -DCMAKE_INSTALL_PREFIX=/home/dynepanch/ros2_ws3/ros2_ws/install/person_msgs
